Tensile/KernelWriterAssembly.py

```diff
@@ -2148,7 +2195,8 @@ def functionSignature(self, kernel ):
       cptSize = "4"
       cptAlign = "4"
       cptByte  = 4
-      cptValueType = "F32"
+      if globalParameters["CodeObjectVersion"] == "V2": cptValueType = "F32"
+      if globalParameters["CodeObjectVersion"] == "V3": cptValueType = "f32"
```

Comment:
Could be simplified with a toUpper() or toLower() at the end, although this section looks like it should be replaced with more fields in DataType.

Reply:
Agreed.  I modified V3 metadata following @amcamd's V2 metadata coding style to help ease the review.  We'll simplify it later.  Thanks.